articles/concepts/multiple-qubits.md

```diff
@@ -42,7 +42,7 @@ For example, there are no states $\psi=\begin{bmatrix} \alpha \\\\  \beta \end{b
 
 $$\psi\otimes \phi = \begin{bmatrix} 1/\sqrt{2} \\\\  0 \\\\  0 \\\\  1/\sqrt{2} \end{bmatrix}.$$ 
 
-Such a two-qubit state, which cannot be written as the tensor product of single-qubit states, is called an "entangled state"; the two qubits are said to be [*entangled*](https://en.wikipedia.org/wiki/Quantum_entanglement).  Loosely speaking, because the quantum state cannot be thought of as a tensor product of single qubit states, the information that the state holds is not confined to either of the qubits individually.  Rather, the information is stored non-locally in the correlations between the two states.  This non-locality of information is one of the major distinguishing features of quantum computing over classical computing and is essential for a number of quantum protocols including [quantum teleportation](https://github.com/Microsoft/Quantum/tree/master/Samples/Teleportation) and [quantum error correction](https://docs.microsoft.com/quantum/libraries/error-correction?view=qsharp-preview).
```

Comment:
The error correction link shouldn't be to an explicit URL at all, I think. Can we change that to be an `xref:` link to the UID for the error correction page?